tests/test_partial.py

```diff
@@ -0,0 +1,69 @@
+"""
+    Pytest Inmanta LSM
+
+    :copyright: 2022 Inmanta
+    :contact: code@inmanta.com
+    :license: Inmanta EULA
+"""
+import pytest
+import utils
+import versions
+
+# skip as early as possible so the rest of this module does not need to be conservative with imports
+if not versions.SUPPORTS_PARTIAL_COMPILE:
+    pytest.skip(
+        "Skipping partial compile tests for inmanta-lsm<2.3.",
+        allow_module_level=True,
+    )
+
+from collections import abc
+
+from inmanta import env
+
+
+@pytest.fixture(scope="session")
+def test_partial_env(pytestconfig) -> abc.Iterator[env.VirtualEnv]:
+    """
+    Yields a Python environment with test_partial installed in it.
+    """
+    with utils.module_v2_venv(pytestconfig.rootpath / "examples" / "test-partial") as venv:
+        yield venv
+
+
+@pytest.fixture(scope="function")
+def testmodulev2_venv_active(
```

Comment:
I'm probably nitpicking here, but isn't it a bad idea to name this fixture and the one above "test..." since they are not actual tests ?

Reply:
I don't mind nitpicking. I changed the names.